docs/instant.md

```diff
@@ -508,7 +508,7 @@ epoch.toZonedDateTimeISO('UTC').until(
   - `roundingIncrement` (number): The granularity to round to, of the unit given by `smallestUnit`.
     The default is 1.
   - `roundingMode` (string): How to handle the remainder, if rounding.
-    Valid values are `'halfExpand'`, `'ceil'`, `'trunc'`, and `'floor'`.
+    Valid values are `'ceil'`, `'floor'`, `'expand'`, `'trunc'`, `'halfCeil'`, `'halfFloor'`, `'halfExpand'`, `'halfTrunc'`, and `'halfEven'`.
```

Comment:
I may be not reading carefully enough, but should we direct the user to look at the docs for `round` to know what each of these values mean?

Reply:
It's not here in `since`, but it is in `until`, and the text of `since` refers the reader to `until`.